dev-container-feature/src/dapr-cli/install.sh

```diff
@@ -0,0 +1,30 @@
+#!/bin/sh
+set -e
+
+echo "Activating feature 'dapr-cli'"
+
+get_latest_release() {
+  curl --silent "https://api.github.com/repos/dapr/cli/releases/latest" |
+  grep '"tag_name":' | sed -E "s/.*\"v([^\"]+)\".*/\1/"
+}
+
+VERSION=${VERSION:-"latest"}
+
+if [ "${VERSION}" = "latest" ]; then
+  VERSION=$(get_latest_release)
+fi
+
+curl -SsL https://github.com/dapr/cli/releases/download/v"${VERSION}"/dapr_linux_amd64.tar.gz | \
+     sudo tar -zx -C /usr/local/bin dapr
+
+dapr --version
+
+## Write bash completion code to a file and source if from .bash_profile
+mkdir -p $_REMOTE_USER_HOME/.dapr
+dapr completion bash >  $_REMOTE_USER_HOME/.dapr/completion.bash.inc
+printf "
+## dapr shell completion
+source '$_REMOTE_USER_HOME/.dapr/completion.bash.inc'
+" >> $_REMOTE_USER_HOME/.bashrc
```

Comment:
Should we add this to the `.zshrc` too (if it exists)?

Reply:
I tried adding to `.zshrc` but I don't get tab completion in zsh. I also tried running `source <(dapr completion zsh)` as per the output from `dapr completion` but still don't get completions.

I'm not really familiar with zsh - how big a deal is this for getting the initial PR merged?

Reply:
That's ok, it's just for shell completion anyways.